### PR TITLE
[WIP] Extensions page and tweaks to website

### PR DESCRIPTION
**Pull request by sjelfull**
## Description

Adding extensions page + fixes and tweaks to website

### CODE REVIEW

File: packages/website/ts/components/footer.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ const linkRows: LinkRows[] = [
         links: [
             { url: WebsitePaths.Instant, text: '0x Instant' },
             { url: WebsitePaths.LaunchKit, text: '0x Launch Kit' },
+            { url: WebsitePaths.Extensions, text: '0x Extensions' },
```
Comment (chriskalani) [inline packages/website/ts/components/footer.tsx]: We can just call these "Extensions" no 0x necessary

File: packages/website/ts/components/dropdowns/dropdown_products.tsx [hunk 2]
```diff
@@ -17,6 +17,11 @@ const navData = [
         description: 'Build on the 0x protocol',
         url: WebsitePaths.LaunchKit,
     },
+    {
+        title: '0x Extensions',
```
Comment (chriskalani) [inline packages/website/ts/components/dropdowns/dropdown_products.tsx]: We can just call it "Extensions".

File: packages/website/ts/pages/about/team.tsx [hunk 3]
```diff
@@ -191,7 +191,16 @@ const advisors: TeamMember[] = [
 export const NextAboutTeam = () => (
     <AboutPageLayout
         title="We are a global, growing team"
-        description="We are a distributed team with backgrounds in engineering, academic research, business, and design. The 0x Core Team is passionate about accelerating the adoption decentralized technology and believe in its potential to be an equalizing force in the world. Join us and do the most impactful work of your life."
+        description={
+            <>
```
Comment (fragosti) [inline packages/website/ts/pages/about/team.tsx]: nit: redundant fragment?

File: packages/website/ts/utils/filter_props.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+                if (allowedProps.includes(key)) {
+                    /* tslint:disable:no-any */
+                    filteredProps[key] = props[key];
+                }
+            }
+
+            /* tslint:disable:no-any */
+            return filteredProps;
+        }
+        public render(): React.ReactNode {
```
Comment (fragosti) [inline packages/website/ts/utils/filter_props.tsx]: would prefer if this were implemented as a render prop component and not a higher order component.

File: packages/website/ts/pages/extensions.tsx [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+    },
+    {
+        icon: 'whitelistFilter',
+        title: 'Whitelist Filter',
+        description: `Restrict access to your relayer with a Whitelist of approved traders. Bring your own list of addresses, or use Wyre's KYC list for free.`,
+        links: [
+            {
+                text: 'Learn More',
+                url:
+                    'https://github.com/0xProject/0x-protocol-specification/blob/master/v2/v2-specification.md#whitelist',
```
Comment (abandeali1) [inline packages/website/ts/pages/extensions.tsx]: This should link to: https://github.com/0xProject/0x-monorepo/tree/development/contracts/extensions/contracts/src/BalanceThresholdFilter

**Pull request merged**